core/gesture.ts

```diff
@@ -1235,9 +1230,6 @@ export class Gesture {
    * @internal
    */
   getInsertionMarkers(): BlockSvg[] {
```

Comment:
I'm probably missing context, but: why does this function continue to exist but always return an empty list? Will you re-implement it later?

Reply:
Mmm I left this like this because I wasn't quite sure what to do with it, since the draggable doesn't have APIs for getting insertion markers, cuz that's super specific to blocks. But I think I can reimplements this by just walking the workspace and checking for any blocks that are flagged as insertion markers. I'll check where this is getting called from and whether that's going to be a performance killer or not.

Reply:
So it turns out that `getInsertionMarkers` is only called from `workspace.render`, so I just moved the relevant logic into there. Performance impact shouldn't be an issue, because we don't actually call that method. And if people are calling it and run into problems, I can give them more performant ways of triggering rerenders.